src/Juvix/Compiler/Casm/Pretty/Base.hs

```diff
@@ -112,25 +112,36 @@ instance PrettyCode InstrExtraBinop where
     incAp <- ppIncAp _instrExtraBinopIncAp
     return $ r <+> Str.equal <+> v1 <+> op <+> v2 <> incAp
 
+ppRel :: Bool -> RValue -> Sem r (Doc Ann)
```

Comment:
should this be a non-monadic function instead? (it is fine to leave it as it is if it is likely that we'll need to read the Options from here in the future)

Reply:
let's leave it as it is